### DISCUSSION TITLE
my custom domain is not work

### DISCUSSION THREAD

**Discussion by Zeng1998** [vercel/vercel#6273] [Help]
- The project can be accessed through the original domain.
- I bought a domain in namesilo and do nothing else
- At "Project Settings > Domains", I added a domain like  `xxxx.top`. It recommended me to add `www.xxxx.top` and redirect `xxxx.top` to `www.xxxx.top`. I do this.
- Now I can access the **original domain**, and the **www.xxxx.top**, but the xxxx.top show me "This site can’t be reached....ERR_CONNECTION_REFUSED"

How can I do?

**Comment by fu-sen** [comment]
If your domain reference was immediately after setting up your name server, you still have to wait. The name server has a cache (TTL).
If it hasn't changed after a while, please check the name server again.

It is useful to look up a server reference with `dig` or` nslookup`.

**Reply by fu-sen** [reply]
I also managed the domain for NameSilo. If you are using this name server, the default TTL is 3600 (1 hour).
